ui/app/controllers/variables/variable.js

```diff
@@ -3,12 +3,14 @@ import Controller from '@ember/controller';
 export default class VariablesVariableController extends Controller {
   get breadcrumbs() {
     let crumbs = [];
-    this.params.path.split('/').reduce((m, n) => {
+    let id = decodeURI(this.params.id.split('@').slice(0, -1).join('@')); // remove namespace
+    let namespace = this.params.id.split('@').slice(-1)[0];
+    id.split('/').reduce((m, n) => {
```

Comment:
`question`: does this handle jobs that have `/` in their name?

Reply:
Hi @ChaiWithJai — that's specifically what this handles (guessing you mean variables).

A `/` in a variable name is the thing that's being split on here, and implies a folder-like hierarchy.